### PR TITLE
solve footer overlaping 565

### LINKED ISSUE #565: footer is overlapping guide.hbs.
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

**Expected behavior:**

No overlapping.
**Steps to reproduce:**
go to http://badgeyay.com/#/guide and scroll down to the bottom. You will see overlapping of footer on guide.hbs template.

**Do you Want to work on it ?**
No, a beginner can take up this issue.

### PR DESCRIPTION
**Pull request by avsingh999**
Fixes #565 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-
    update frontend/app/templates/guide.hbs
-
   update frontend/public/css/style.css

### CODE REVIEW

File: frontend/public/css/style.css [hunk 1]
```diff
@@ -6,6 +6,12 @@
     top:3px;
 }
 
+.foooter {
```
Comment (vaibhavsingh97) [inline frontend/public/css/style.css]: change `.foooter` w/ `.footer`

File: frontend/public/css/style.css [hunk 2]
```diff
@@ -6,6 +6,12 @@
...
+    margin-top: 10px;
```
Comment (vaibhavsingh97) [inline frontend/public/css/style.css]: This can be re written as `margin: 10px 0 20px 0;`

**Comment by vaibhavsingh97** [comment]
Awesome! Congratulations on your first PR. One commit is preferred for such small PR. Please read [FOSSASIA best practices](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia/). Please include screenshots in your PR message.

File: frontend/public/css/style.css (hunk 1 above)
Comment (gabru-md) [reply frontend/public/css/style.css]: @vaibhavsingh97  what?

**Comment by gabru-md** [comment]
@vaibhavsingh97 I suppose changes can be viewed on the surge deployment, so there is no need to post screenshots here.
Link to the changes : [pr-601-fossasia-badgeyay.surge.sh](pr-601-fossasia-badgeyay.surge.sh)

File: frontend/public/css/style.css (hunk 1 above)
Comment (gabru-md) [reply frontend/public/css/style.css]: Sorry! 
Did not notice the 3 Os

**Pull request merged**